### PR TITLE
update dj onboarding emails

### PR DESCRIPTION
**Pull request by mcfiredrill**

### CODE REVIEW

File: app/views/dj_account_mailer/welcome_email.html.erb [hunk 1]
```diff
@@ -8,23 +8,25 @@ Your account on <%= @radio.name %> has been created.
 
 <p>
 Your username is <%= @user.username %>.
-Your password is <%= @password %>. You can change your password <%== link_to "here.", edit_user_registration_url(@user) %>
+Your password is <%= @password %>.
```
Comment (dudethatbe) [inline app/views/dj_account_mailer/welcome_email.html.erb:11]: do we really need to include the password in the email?

**Review by dudethatbe — CHANGES_REQUESTED** [review]
seems good to me. only one comment about the password being stored in the email. I also don't understand why there has to be a HTML and text version of the same thing. sure does seem like it makes it difficult to maintain if there's two identical(ish) files

**Comment by mcfiredrill** [comment]
> I also don't understand why there has to be a HTML and text version of the same thing. sure does seem like it makes it difficult to maintain if there's two identical(ish) files

Yeah it's pretty annoying. I wonder how many people really need plain text only emails anyway.

File: app/views/dj_account_mailer/welcome_email.html.erb (hunk 1 above)
Comment (mcfiredrill) [reply app/views/dj_account_mailer/welcome_email.html.erb:11]: Definitely not if we are converting the account from listener to dj.
But
If it's a brand new account they haven't set a password yet.

Maybe we can just make them use the pw reset immediately instead?

Comment (mcfiredrill) [reply app/views/dj_account_mailer/welcome_email.html.erb]: updated to only show PW for completely new accounts @dudethatbe

**Review by dudethatbe — APPROVED** [review]

**Pull request merged**